qiskit/providers/ibmq/ibmqfactory.py

```diff
@@ -63,13 +86,17 @@ def enable_account(
         Args:
             token: IBM Quantum Experience token.
             url: URL for the IBM Quantum Experience authentication server.
+            hub: Name of the hub to filter for.
+            group: Name of the group to filter for.
+            project: Name of the project to filter for.
             **kwargs: Additional settings for the connection:
 
                 * proxies (dict): proxy configuration.
                 * verify (bool): verify the server's TLS certificate.
 
         Returns:
-            The provider for the default open access project.
+            The provider for the default open access project, or a default provider,
+            if specified by `hub`, `group`, `project`.
```

Comment:
How about
```
If `hub`, `group`, and `project` are specified, the corresponding provider is returned. Otherwise the provider for the open access project is returned.
```